portal/templates/redesign/help-and-support_new.html

```diff
@@ -34,28 +54,147 @@ <h1 class="banner__text--primary">Help and support</h1>
-        <p class="body-text"><strong>+ I want to sign up as a Teacher. Why am I asked for my 'professional' email address when registering?</strong></p>
-        <p class="body-text"><strong>+ What should I do when my students leave my school or club?</strong></p>
-        <p class="body-text"><strong>+ What is an independent student?</strong></p>
-        <p class="body-text"><strong>+ What if a student moves to another school? Can they keep their progress?</strong></p>
-        <p class="body-text"><strong>+ What do I need to do to allow an independent student to join my class or school?</strong></p>
-        <p class="body-text"><strong>+ I've set up my students. How do they log in?</strong></p>
-        <p class="body-text"><strong>+ As a student in a school or club, who can I share my custom created levels with?</strong></p>
-        <p class="body-text"><strong>+ As a teacher, who can I share my custom created levels with?</strong></p>
-        <p class="body-text"><strong>+ How can I control the levels my students create?</strong></p>
-        <p class="body-text"><strong>+ I am an independent student. Why can't I share my custom created levels?</strong></p>
+<div class="faqs background col-sm-8 col-center">
+    <h3>FAQs</h3>
+    <div>
+        <p class="body-text"><strong> I am a school student and have forgotten my password. How do I reset it?</strong></p>
+        <div>
+            <p>Your teacher is the only person who can update or reset your password, so you should speak to them.</p>
+        </div>
+    </div>
+    <div>
+        <p class="body-text"><strong> I am a teacher and have forgotten my password. How do I reset it?</strong></p>
+        <div>
+            <p>Go to the <a href="{% url 'teach' %}">Teach</a> page and select the forgotten password link at the bottom of the log in box to be sent</strong></p>
+        </div>
+    </div>
+
+    <div>
+        <p class="body-text"><strong> I am an independent student and have forgotten my password. How do I reset it?</strong></p>
+        <div>
+            <p>Go to the <a href="{% url 'play' %}">Play</a> page, go to the independent students' login box and select the forgotten password link at the bottom of the log in box to be sent a link to reset your password.</p>
+        </div>
+    </div>
+
+    <div>
+        <p class="body-text"><strong> I'm having problems choosing an acceptable password for my account. What am I missing?</strong></p>
+        <div>
+            <p>Passwords for Teachers must be at least 8 characters long and contain numbers and both lowercase and uppercase letters.
+                <br>Passwords for Students can use any combination of characters but must be at least 6 characters long.</p>
+            <p>If you are having any difficulties with setting your password, please do not hesitate to <a href="{% url 'contact' %}">contact us</a> for support.</p>
+        </div>
+    </div>
+
+
+        <div>
```

Comment:
Make sure indentation is respected here. This makes it easier to check which tags open/close and where, and therefore it's easier to spot if a closing tag is missing.